### DISCUSSION TITLE
How to use window in Next JS

### DISCUSSION THREAD

**Discussion by zekhoi** [vercel/next.js#23094] [Help]
I got an error when using `window`

`Server Error
ReferenceError: window is not defined
This error happened while generating the page. Any console logs will be displayed in the terminal window.`

is there any other way to use it? i want to change the nav background when scrolling down

**Comment by larbisahli** [comment]
Use this:

`let isMobileView;
if (typeof window !== 'undefined'){
  isMobileView = window.innerWidth < 735;
}`
